CHANGELOG.md

```diff
@@ -3,13 +3,25 @@ Changelog
 
 Unreleased
 ----------
+
+### Breaking
+* Removed the hide option from navigation.config
+* Removed the theme imports option from site.conf
+* Removed extension config option from site.conf
+* The Browser router is now used instead of the hash router.
+
 ### Changed
 * Updated fs-extra dependency to ^8.0.0
 * Updated babel-eslint dev-dependency to ^10.0.1
 * Added mixin pattern rule in package.json.
 
-### Fixed
-* Fixed the console error which occured when ESC was pressed in the search filter.
+### Added
+* consumed new terra-application and terra-application-navigation packages
+* add devToolbar
```

Comment:
add devToolbar to display a development toolbar that allows switching between locales and themes.